### PR TITLE
Feature/498 terraform fix integration event listener

### PR DESCRIPTION
**Pull request by ZavezX**
## Description

- Use event hub sender connection string

## References

* #498

### CODE REVIEW

**Review by mknic — APPROVED** [review]
LGTM

File: .github/workflows/infrastructure-cd.yml [hunk 1]
```diff
@@ -98,6 +74,32 @@ jobs:
...
+          secrets: 'SHARED-RESOURCES-DB-ADMIN-NAME,SHARED-RESOURCES-DB-ADMIN-PASSWORD,DELTA-LAKE-STORAGE-ACCOUNT-KEY,DELTA-LAKE-STORAGE-ACCOUNT-NAME,DELTA-LAKE-CONTAINER-NAME,DELTA-LAKE-AGGREGATION-MASTER-DATA-BLOB-NAME,SHARED-RESOURCES--SB-INTEGRATIONEVENTS-LISTEN-CONNECTION-STRING'  # comma separated list of secret keys that need to be fetched from the Key Vault 
+        id: sharedGetSecretAction
+        
+      # get keys from aggregation keyvault
+      - name: Azure key vault - Get Secrets
+        uses: Azure/get-keyvault-secrets@v1.2
+        with:
+          keyvault: "kvaggregationsendku"
+          secrets: 'aggregation-evh-listening-key,aggregation-evh-sender-key'  # comma separated list of secret keys that need to be fetched from the Key Vault 
+        id: myGetSecretAction
```
Comment (jonasdmoeller) [inline .github/workflows/infrastructure-cd.yml:91]: "myGetSecretAction" is not a good name. Maybe "aggregationGetSecretAction" is better?

**Review by jonasdmoeller — CHANGES_REQUESTED** [review]

File: .github/workflows/infrastructure-cd.yml [hunk 2]
```diff
@@ -98,6 +74,32 @@ jobs:
...
+          keyvault: "kvsharedresendku"
+          secrets: 'SHARED-RESOURCES-DB-ADMIN-NAME,SHARED-RESOURCES-DB-ADMIN-PASSWORD,DELTA-LAKE-STORAGE-ACCOUNT-KEY,DELTA-LAKE-STORAGE-ACCOUNT-NAME,DELTA-LAKE-CONTAINER-NAME,DELTA-LAKE-AGGREGATION-MASTER-DATA-BLOB-NAME,SHARED-RESOURCES--SB-INTEGRATIONEVENTS-LISTEN-CONNECTION-STRING'  # comma separated list of secret keys that need to be fetched from the Key Vault 
+        id: sharedGetSecretAction
+        
+      # get keys from aggregation keyvault
+      - name: Azure key vault - Get Secrets
+        uses: Azure/get-keyvault-secrets@v1.2
+        with:
+          keyvault: "kvaggregationsendku"
+          secrets: 'aggregation-evh-listening-key,aggregation-evh-sender-key'  # comma separated list of secret keys that need to be fetched from the Key Vault
```
Comment (ZavezX) [inline .github/workflows/infrastructure-cd.yml:90]:
```suggestion
          secrets: 'aggregation-evh-listening-key'  # comma separated list of secret keys that need to be fetched from the Key Vault 
```

File: .github/workflows/infrastructure-cd.yml (hunk 1 above)
Comment (ZavezX) [reply .github/workflows/infrastructure-cd.yml:91]: The section is taken dircetly from an example online, but I did not add it, I only changed the position in the file. Let's just keep the naming as is.

**Review by johevemi — APPROVED** [review]

**Review by jonasdmoeller — APPROVED** [review]

**Pull request merged**